### PR TITLE
Create a spec skeleton with a few filled out parts.

### PR DESCRIPTION
**Pull request by rayankans**
Adds a spec skeleton for the Contact Picker API. The information
included in the proposal was included, with a few minor changes, and a
section on the picker UI.

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+# Contact Picker # {#contact-picker}
+
+<div algorithm>
+
+A contact picker MUST be initiated with |options|, an instance of {{ContactsSelectOptions}}.
+
+To <dfn>launch</dfn> a contact picker, the user agent MUST present a user interface that follows these rules:
+
+* The UI MUST prominently display the [=browsing context=]'s [=origin=].
+* The UI MUST make it clear which `properties` of the contact will be shared.
```
Comment (beverloo) [inline spec/index.bs]: SHOULD? I agree with requiring origin, but this kind of is a UI choice.

File: spec/index.bs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+* The UI MUST provide an option to cancel/return without sharing any contacts.
+* If |options|' `multiple` member is false, only one contact should be pickable.
```
Comment (beverloo) [inline spec/index.bs]: should -> MUST?

File: spec/index.bs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:dfn; for:/; text:browsing context
+spec:html; type:dfn; for:/; text:origin
+</pre>
+
+# Introduction # {#intro}
+
+Contact pickers are frequently seen in native mobile applications for a variety of use cases, and in various desktop applications such as e-mail clients and calendars.
```
[the remark this answers is not in the document]
Comment (rayankans) [reply spec/index.bs]: Makes sense, went for a 100-char limit

[the remark this answers is not in the document]
Comment (rayankans) [reply spec/index.bs]: I briefly touched on that in the new commit, but added a TODO to fully flesh out the details. I'd prefer this pull request to focus on the initial structure of things.

File: spec/index.bs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+* Social networks could use contact information to bootstrap a user's social graph. This is particularly important for emerging markets, where users might prefer a PWA over a native app due to storage constraints.
+* An e-mail application could allow the user to select the recipients for a message without needing their own address book.
+
+## Example ## {#example}
+
+TODO: Add examples.
+
+# Privacy Considerations # {#privacy}
+
+Exposing contact information has a clear privacy impact, in terms of exposing PII of uninvolved parties. A picker model is enforced so that the user agent can offer a user experience that makes it clear what information is going to be shared with the website.
```
[the remark this answers is not in the document]
Comment (rayankans) [reply spec/index.bs]: Do you mean what PII is, or  what the PII fields in this case are?

File: spec/index.bs (hunk 1 above)
Comment (rayankans) [reply spec/index.bs]: There's precedent for enforcing things in the UI. For example the Background Fetch spec enforces that the origin must be shown in the notification. I think enforcing it has benefits, WDYT?

File: spec/index.bs (hunk 2 above)
Comment (rayankans) [reply spec/index.bs]: It makes sense, but I can't think of a good reason to enforce this. I can be convinced though

**Pull request merged**